circuits/cpp/barretenberg/cpp/src/barretenberg/proof_system/relations/auxiliary_relation.hpp

```diff
@@ -20,8 +18,8 @@ template <typename FF> class AuxiliaryRelationBase {
     static constexpr size_t LEN_4 = 6; // RAM consistency sub-relation 1
```

Comment:
I'd rather expose an array but there's no easy way to unpack an array to a parameter pack.